### ISSUE TITLE
Buzz controller emulation?

### ISSUE THREAD

**Issue by Haxorzz** [Alby1987/BuzzMultiBuzzersPlugin#5]
My original buzz controllers for the ps2 got lost, and now theres no way to play my favourite game with some of my friends, is there a fix for this? Or anyway to emulate buzz controllers using other input methods?

**Comment by FKCrew** [comment]
Put the configuration like this 

![image](https://user-images.githubusercontent.com/84207879/122812124-93869280-d2d1-11eb-8e19-1fa8b9964b52.png)

**Comment by Haxorzz** [comment]
Does this really work? If so this is awesome! What are the buttons bound to?

**Comment by FKCrew** [comment]
Yeah, it works. The buttons are bound to one real Buzz! Controller.

**Comment by Leptys** [comment]
> Put the configuration like this
> 
> ![image](https://user-images.githubusercontent.com/84207879/122812124-93869280-d2d1-11eb-8e19-1fa8b9964b52.png)

how the hell did u do that? 
Like, what section of what emulator?

**Comment by FKCrew** [comment]
![image](https://user-images.githubusercontent.com/84207879/124366508-a80f4700-dc50-11eb-8001-a143d8543f6f.png)

USB > Buzz MultiBuzzers > Configure :)

**Comment by Haxorzz** [comment]
This post was made because I lost my buzz controllers. I wanted them to be binded to keys.
